internal/server/httpServer.go

```diff
@@ -2,20 +2,28 @@ package server
 
 import (
 	"fmt"
+	"log/slog"
+
 	"github.com/fasdalf/train-go-musthave-metrics/internal/server/handlers"
+
 	"github.com/gin-gonic/gin"
 	slogGin "github.com/samber/slog-gin"
-	"log/slog"
 )
 
-func NewHTTPEngine(ms handlers.Storage) *gin.Engine {
+func NewHTTPEngine(ms handlers.Storage, fs handlers.FileStorage) *gin.Engine {
 	ginCore := gin.New()
 	ginCore.Use(slogGin.New(slog.Default()))
 	ginCore.Use(gin.Recovery())
+	// import "github.com/gin-contrib/gzip"
+	//ginCore.Use(gzip.Gzip(gzip.DefaultCompression))
+	// uses internal/server/handlers/gzip.go + internal/server/handlers/gzipCompressionHandler.go
+	ginCore.Use(handlers.GzipCompressionHandler)
 
 	// check with and w/o trailing slash
 	ginCore.GET("/", gin.WrapF(handlers.NewIndexHandler(ms)))
 	ginCore.GET(fmt.Sprintf("/value/:%s/:%s", handlers.URLParamType, handlers.URLParamName), handlers.NewViewStatsHandler(ms))
+	ginCore.POST("/value/", handlers.CheckMetricExistenceHandler(ms), handlers.MetricValueResponseHandler(ms))
```

Comment:
Мне кажется trailing slash можно опустить.

```suggestion
	ginCore.POST("/value", handlers.CheckMetricExistenceHandler(ms), handlers.MetricValueResponseHandler(ms))
```

Reply:
оказалось можно

Reply:
++